src/desktop/docks/titlewidget.cpp

```diff
@@ -85,6 +85,13 @@ void TitleWidget::addGlobalDockActions(const QList<QAction *> &actions)
 	m_menu->addActions(actions);
 }
 
+void TitleWidget::setKeepButtonSpace(bool keepButtonSpace)
```

Comment:
I have not run a build of this version yet to check what this is doing, is it trying to keep empty space in the title where the dock buttons are when the docks aren’t locked, instead of allowing the contents of the dock title to expand to fill all available space? If so, I deleted that from my fork, because making the buttons not collapse just means that space is wasted the 99+% of the time you aren’t rearranging your layout.

Reply:
It's only used by the brush dock, which wants to center its tool slots. All other docks will fill up the title bar space as you describe.